src/main/java/com/nike/cerberus/command/core/GenerateCertificateFilesCommandParametersDelegate.java

```diff
@@ -0,0 +1,199 @@
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.command.core;
+
+import com.beust.jcommander.Parameter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GenerateCertificateFilesCommandParametersDelegate {
+    
+    public static final String BASE_DOMAIN_LONG_ARG = "--base-domain";
+    public static final String EDGE_DOMAIN_NAME_OVERRIDE_LONG_ARG = "--edge-domain-name-override";
+    public static final String ORIGIN_DOMAIN_NAME_OVERRIDE_LONG_ARG = "--origin-domain-name-override";
+    public static final String LOAD_BALANCER_DOMAIN_NAME_OVERRIDE_LONG_ARG = "--load-balancer-domain-name-override";
+    public static final String HOSTED_ZONE_ID_LONG_ARG = "--hosted-zone-id";
+    public static final String SUBJECT_ALT_NAME_LONG_ARG = "--subject-alternative-name";
+    public static final String ENABLE_LE_CERTFIX_LONG_ARG = "--enable-letsecrypt-certfix";
+    public static final String CERT_FOLDER_LONG_ARG = "--local-certificate-directory";
+    public static final String ACME_API_LONG_ARG = "--acme-api-url";
+    public static final String CONTACT_EMAIL_LONG_ARG = "--contact-email";
+    public static final String NO_TTY_LONG_ARG = "--no-tty";
+    public static final String ACCEPT_ACME_TOS = "--no-tty-force-acme-tos-accept";
+
+    public static final String LETS_ENCRYPT_ACME_API_URI = "acme://letsencrypt.org";
+
+    @Parameter(
+            names = {
+                    BASE_DOMAIN_LONG_ARG
+            },
+            description = "The base domain for the environment that this command will use to generate the following " +
+                    "subject name {env}.{base-domain} and with the following subject alternative name {env}.{region}.{base-domain}\n" +
+                    "ex: cerberus -e demo -r us-west-2 generate-certificates --base-domain cerberus.example would make a cert for demo.cerberus.example " +
+                    "with a sans of demo.us-west-2.cerberus.example so that we can create a CNAME record for " +
+                    "demo.cerberus.example that will point to an ALB in us-west-2 with a CNAME record of " +
+                    "demo.us-west-2.cerberus.example and the cert will be valid for both"
+    )
+    private String baseDomainName;
+    
+    @Parameter(
+            names = {
+                    EDGE_DOMAIN_NAME_OVERRIDE_LONG_ARG
+            },
+            description = "This command uses {environment}.{base-domain} as the common name, override it with this option"
+    )
+    private String edgeDomainNameOverride;
+    
+    @Parameter(
+            names = {
+                    ORIGIN_DOMAIN_NAME_OVERRIDE_LONG_ARG
+            },
+            description = "origin domain name defaults to origin.{environment-name}.{base-domain}, " +
```

Comment:
nitpick: All of these descriptions should start with a capital letter

Reply:
- [ ] DONE